libs/go/libp2p_node/dht/dhtpeer/dhtpeer.go

```diff
@@ -629,7 +629,6 @@ func (dhtPeer *DHTPeer) handleDelegateService(ready *sync.WaitGroup) {
 				go dhtPeer.handleNewDelegationConnection(conn)
 			}
 		case <-dhtPeer.closing:
-			break
```

Comment:
looks like it will never exit `for` loop

Reply:
yes sorry, missed it, will fix